src/domain/entities/Template.ts

```diff
@@ -312,6 +314,7 @@ export function setDataEntrySheet(dataSource: RowDataSource, sheets: SheetE[]):
                       longitude: set(dataSource.coordinates.longitude),
                   }
                 : undefined,
+            geometry: dataSource.geometry ? dataSource.geometry : undefined,
```

Comment:
`a ? a : b` -> `a || b`. But also, `RowDataSource.geometry?: CellRef | ValueRef<string> | ColumnRef | undefined
`, so the only falsy value is undefined. All in all, it seems `dataSource.geometry` should be enough.